packages/server/lib/util/uri.js

```diff
@@ -1,44 +1,53 @@
-// TODO: This file was created by bulk-decaffeinate.
-// Sanity-check the conversion and remove this comment.
-/*
- * decaffeinate suggestions:
- * DS102: Remove unnecessary code created because of implicit returns
- * Full docs: https://github.com/decaffeinate/decaffeinate/blob/master/docs/suggestions.md
- */
+const _ = require('lodash')
 const url = require('url')
 
-const DEFAULT_PORTS = ['443', '80']
+const DEFAULT_PORTS = {
+  https: '443',
+  http: '80',
+}
 
 const stripProtocolAndDefaultPorts = function (urlToCheck) {
-  //# grab host which is 'hostname:port' only
+  // grab host which is 'hostname:port' only
   const { host, hostname, port } = url.parse(urlToCheck)
 
-  //# if we have a default port for 80 or 443
-  //# then just return the hostname
-  if (DEFAULT_PORTS.includes(port)) {
+  // if we have a default port for 80 or 443
+  // then just return the hostname
+  if (_.values(DEFAULT_PORTS).includes(port)) {
```

Comment:
Since we use the values here twice (and they are a constant) we should set another constant at the top next to `DEFAULT_PORTS` called like `DEFAULT_PORT_VALUES`